docs/docs/Concepts/concepts-flows.md

```diff
@@ -3,23 +3,34 @@ title: Flows
 slug: /concepts-flows
 ---
 
-Flows in Langflow are fully serializable and can be saved and loaded from the file system. In this guide, we'll explore how to import and export flows.
+import Icon from "@site/src/components/icon";
 
-## Import Flow
+Flows in Langflow are fully serializable and can be saved and loaded from the file system. In this guide, we'll explore how to import and export flows.
 
-If you've already got a Langflow JSON file, import it into Langflow by clicking on the project name and choosing **Import Flow**.
+## Import flow
 
-![Import Flow](../../static/img/flows/import.gif)
+If you already have a Langflow JSON file on your local machine, from the **Projects** page, click <Icon name="Upload" aria-hidden="true"/> **Upload a flow**.
 
 Once imported, your flow is ready to use.
 
 :::tip
 You can drag and drop Langflow JSON files directly from your file system into the Langflow window to import a flow, even into the initial Langflow splash screen.
 :::
 
-## Export Flow
+## Export flow
+
+To **Export** your flow, in the **Playground**, click **Share**, and then click **Export**.
+
+Select **Save with my API keys** to save the flow with any **Global variables** included.
+
+:::important
```

Comment:
```suggestion
:::warning
```

I would increase this to warning because it potentially exposes the literal key.